app/views/datasets/_breadcrumb.html.erb

```diff
@@ -4,19 +4,21 @@
     <div class="column-full">
       <div class="datagov_breadcrumb">
         <nav aria-label="Breadcrumb">
-          <li>
-            <%= link_to t('.home'), root_path %>
-          </li>
-          <li>
-            <% if @referer_query.nil? %>
-              <%= @dataset.organisation.title %>
-            <% else %>
-              <%= link_to t('.search'), "/search?#{@referrer}" %>
-            <% end %>
-          </li>
-          <li aria-current="page">
-            <%= shorten_title(@dataset.title) %>
-          </li>
+          <ul>
```

Comment:
I think this should be an ordered list (`ol`) as there is natural ordering of the breadcrumb elements.

This is the approach adopted in https://github.com/alphagov/govuk-frontend/blob/master/src/breadcrumbs/template.njk

Reply:
Yeah I changed it from <ul> to <ol> after reading the W3 guidelines which suggested unless there's a number or similar it should be ol. But on the MDN docs it says different. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol

Is it generally better to use the MDN docs?

Reply:
> As a rule of thumb to determine which one to use, try changing the order of the list items; if the meaning is changed, the <ol> element should be used, else the <ul> is adequate.